elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/DynamicConfigHelpers.java

```diff
@@ -31,11 +31,14 @@
  */
 public class DynamicConfigHelpers {
 
-    private static final String TABLE_CONFIG_PATH = "tables/";
+    private static String TABLE_CONFIG_PATH = "tables/";
     private static final String SECURITY_CONFIG_PATH = "security.hjson";
     private static final String VARIABLE_CONFIG_PATH = "variables.hjson";
     private static final String NEW_LINE = "\n";
 
+    public static void setTableConfigPath(String input) {
```

Comment:
Instead of using statics (which are hard to test), the config path should be passed in as an argument.